friends/src/App.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import { Route } from 'react-router-dom';
+
+import FriendForm from './Components/FriendForm';
+import FriendList from './Components/FriendList';
+import './App.css';
+
+class App extends Component {
+	constructor() {
+		super();
+		this.state = {
+			friends: []
+		};
+	}
+	componentDidMount() {
+		axios
+			.get('http://localhost:5000/friends')
+			.then((response) => {
+				console.log('response:', response);
+				this.setState({ friends: response.data });
+			})
+			.catch((err) => console.log('error:', err));
+	}
+	postFriend = (friend) => {
+		axios
+			.post('http://localhost:5000/friends', friend)
+			.then((response) => {
+				console.log('response', response);
+				this.setState({ friends: response.data });
+			})
+			.catch((err) => console.error('error:', err));
+	};
+	updateFriend = (friend, id) => {
```

Comment:
You have two `updateFriend` functions in your project, but this is the one that's sent down to the button displayed on the screen. Note that you have it taking in the `friend` to be updated and the `id` of the friend.